src/libs/ReportUtils.js

```diff
@@ -1328,20 +1331,12 @@ function isWaitingForIOUActionFromCurrentUser(report) {
         return false;
     }
 
-    const policy = getPolicy(report.policyID);
-    if (policy.type === CONST.POLICY.TYPE.CORPORATE) {
-        // If the report is already settled, there's no action required from any user.
-        if (isSettled(report.reportID)) {
-            return false;
-        }
-
-        // Report is pending approval and the current user is the manager
-        if (isReportManager(report) && !isReportApproved(report)) {
-            return true;
-        }
+    if (report.isUnreadWithMention) {
+        return true;
+    }
 
-        // Current user is an admin and the report has been approved but not settled yet
-        return policy.role === CONST.POLICY.ROLE.ADMIN && isReportApproved(report);
+    if (report.isWaitingForTaskCompleteFromAssignee) {
+        return true;
     }
 
     // Money request waiting for current user to add their credit bank account
```

Comment:
@waterim actually, we've decided to treat this condition like the others. If a report is waiting on bank account, we will add the `hasOutstandingChildRequest` to the parent for the submitter and show the green dot on the parent. So let's remove this condition.

Reply:
@waterim I think we should remove this whole block: 

```
    // Money request waiting for current user to add their credit bank account
    // hasOutstandingIOU will be false if the user paid, but isWaitingOnBankAccount will be true if user don't have a wallet or bank account setup
    if (!report.hasOutstandingIOU && report.isWaitingOnBankAccount && report.ownerAccountID === currentUserAccountID) {
        return true;
    }

Reply:
@waterim I think we should remove this whole block: 

```
    // Money request waiting for current user to add their credit bank account
    // hasOutstandingIOU will be false if the user paid, but isWaitingOnBankAccount will be true if user don't have a wallet or bank account setup
    if (!report.hasOutstandingIOU && report.isWaitingOnBankAccount && report.ownerAccountID === currentUserAccountID) {
        return true;
    }